client/volume_prune.go

```diff
@@ -29,7 +29,7 @@ func (cli *Client) VolumesPrune(ctx context.Context, pruneFilters filters.Args)
 	defer ensureReaderClosed(serverResp)
 
 	if err := json.NewDecoder(serverResp.body).Decode(&report); err != nil {
-		return report, fmt.Errorf("Error retrieving disk usage: %v", err)
+		return report, fmt.Errorf("Error retrieving volume prune report:: %v", err)
```

Comment:
Good catch. I think it is a mistake introduced from this PR. sorry for that. And I am add a pr to fix this